src/components/HotPost/HotPost.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import {Link} from "react-router-dom";
+import Truncatenator from "../../helpers/Truncatenator";
+
+function HotPost({permalink, title, subreddit, subredditIdPrefix, comments, upvotes}) {
+
+    return (
+        <>
+            <article className="hotpost">
+                <h3 className="article-title"><a href={`https://www.reddit.com${permalink}`}>{Truncatenator(title)}</a></h3>
+                <span>
+            <p className="hotpost-subreddit"><Link to={`/subreddit/r/${subreddit}`}>{subredditIdPrefix}</Link></p>
+            <p>Comments {comments} - Upvotes {upvotes}</p></span>
+            </article>
+        </>
```

Comment:
De lege tags kunnen weggelaten worden, omdat het maar één child bevat:
```suggestion
        <article className="hotpost">
            <h3 className="article-title"><a href={`https://www.reddit.com${permalink}`}>{Truncatenator(title)}</a></h3>
            <span>
                <p className="hotpost-subreddit"><Link to={`/subreddit/r/${subreddit}`}>{subredditIdPrefix}</Link></p>
                <p>Comments {comments} - Upvotes {upvotes}</p>
            </span>
        </article>
```